### PR TITLE
HPCC-13202 WU Input files out of date.

### PR DESCRIPTION
**Pull request by GordonSmith**
There is no way to see the contents of a source superfile at the time the WU
was run.

Fixes HPCC-13202

Signed-off-by: Gordon Smith gordon.smith@lexisnexis.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

**Comment by AttilaVamos** [comment]
Automated Smoketest Build: success

**Comment by miguelvazq** [comment]
Looks fine.

File: esp/src/eclwatch/ESPWorkunit.js [hunk 1]
```diff
@@ -122,7 +122,12 @@ define([
         _SourceFilesSetter: function (SourceFiles) {
             var sourceFiles = [];
             for (var i = 0; i < SourceFiles.ECLSourceFile.length; ++i) {
-                sourceFiles.push(ESPResult.Get(lang.mixin({ wu: this.wu, Wuid: this.Wuid }, SourceFiles.ECLSourceFile[i])));
+                sourceFiles.push(ESPResult.Get(lang.mixin({ wu: this.wu, Wuid: this.Wuid, __hpcc_parentName: "" }, SourceFiles.ECLSourceFile[i])));
```
Comment (richardkchapman) [inline esp/src/eclwatch/ESPWorkunit.js]: This looks like it is fixing something else?

Comment (GordonSmith) [inline esp/src/eclwatch/ESPWorkunit.js]: Thats not - its for the root nodes in the tree.

**Pull request merged**